roles/edpm_telemetry/molecule/default/verify.yml

```diff
@@ -59,6 +61,13 @@
         - "Starting node_exporter"
         - "TLS is enabled"
 
+    - name: ensure that Kepler is running
+      become: true
+      ansible.builtin.shell: |
+        podman logs kepler 2>&1 | grep "{{item}}" > /dev/null
+      loop:
+        - "Started Kepler"
+
```

Comment:
Can we just check if the container is running? Do we expect a running container that doesn't work?

Reply:
No, we don't expect such a situation. I have removed this test.